public/books/2021/84k.md

```diff
@@ -15,10 +15,10 @@ After I got used to the technical aspect of the writing, a really engrossing sto
 
 The book is written in a way that quite often ends right in the middle of a train of thought. Reading on a kindle it was hard to work out if it was intentional or if there was a problem with the Kindle edition! After a while I got used to it, and learnt to just to either let go of the sentiment or ignore it.
 
-The story uses time jumping *a lot* but thankfully the book is written in a way that it was easy to know when we had jumped back or forwards through time.
+The story uses time jumping *a lot* ,but thankfully the book is written in a way that it was easy to know when we had jumped back or forwards through time.
```

Comment:
Don't know about English comma rules, but in any case, if there should be a comma, then it should be before the space
```suggestion
The story uses time jumping *a lot*, but thankfully the book is written in a way that it was easy to know when we had jumped back or forwards through time.
```